source/game/kart/KartSuspensionPhysics.hh

```diff
@@ -8,6 +8,13 @@
 
 namespace Kart {
 
+/// @brief Every other kart tire is a mirror of the first. Bikes do not leverage this.
+enum class MirroredTire {
+    NotMirrored,
+    Mirrored,
+    Bike,
+};
```

Comment:
```suggestion
enum class TireType {
    Kart,
    KartReflected,
    Bike,
};
```
I'm not a fan of unscoped enum classes. If this enum class must exist within this header, I would prefer scoping it within `KartSuspensionPhysics`.

Reply:
Not 100% sure if this should be moved to a different header, but for now I scoped it within `KartSuspensionPhysics`